typespec-tests/Setup.ps1

```diff
@@ -1,3 +1,12 @@
+param (
+    # re-build autorest.java
+    [switch] $RebuildJar = $false
+)
+
+if ($RebuildJar) {
+    mvn clean install package -f ../pom.xml -Plocal -Ptsp -DskipTests
+}
+
```

Comment:
For local run.

E.g. `pwsh Setup.ps1 -RebuildJar` will rebuild autorest.java for typespec-java.